### DISCUSSION TITLE
可以支持下AWS DynamoDB 数据源吗?

### DISCUSSION THREAD

**Discussion by yangguocang** [hhyo/Archery#2832] [Ideas]
可以支持下AWS DynamoDB 数据源吗?

**Comment by feiazifeiazi** [comment]
@yangguocang 
看了DynamoDB 介绍，竟然不知道我该什么情况下用它。
能不能给涨涨知识，你们在什么情况下，用这个数据库？


Amazon DynamoDB 是一种无服务器、NoSQL、完全托管的数据库，在任何规模下都具有个位数毫秒的性能。

DynamoDB 满足您克服关系数据库扩展和操作复杂性的需求。DynamoDB 是专门为需要在任何规模下都具有一致性能的运营工作负载而构建和优化的。例如，无论您有 1000 万还是 1 亿用户，DynamoDB 都能为购物车用例提供一致的个位数毫秒级性能。 [于 2012 年推出](https://press.aboutamazon.com/2012/1/amazon-web-services-launches-amazon-dynamodb-a-new-nosql-database-service-designed-for-the-scale-of-the-internet)，DynamoDB 继续帮助您摆脱关系数据库，同时降低成本并提高大规模性能。

各种规模、行业和地区的客户都使用 DynamoDB 构建现代无服务器应用程序，这些应用程序可以从小规模开始，然后在全球范围内扩展。DynamoDB 可扩展以支持几乎任何规模的表，同时提供一致的个位数毫秒级性能和高可用性。
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Introduction.html
